File: app/src/main/java/com/feriantes4dawin/feriavirtualmovil/data/services/UsuarioAPIService.java
package com.feriantes4dawin.feriavirtualmovil.data.services;

import com.feriantes4dawin.feriavirtualmovil.data.network.responses.UsuarioResponse;
import retrofit2.http.*;

public interface UsuarioAPIService {

    @FormUrlEncoded
    @POST("auth/login")
    UsuarioResponse login(

        @Field(value="email")
        String email,

        @Field(value="contrasena")
        String contrasena

    );

    @GET("usuario/{idusuario}")
    UsuarioResponse getUserInfo(

            @Path(value="idusuario")
            Integer personalID

    );

}

File: app/src/main/java/com/feriantes4dawin/feriavirtual/data/services/UsuarioAPIService.kt
package com.feriantes4dawin.feriavirtual.data.services

import com.feriantes4dawin.feriavirtual.data.network.responses.UsuarioResponse
import kotlinx.coroutines.Deferred
import retrofit2.http.*

interface UsuarioAPIService {

    @FormUrlEncoded
    @POST("auth/login")
    fun login(

        @Field(value="email")
        email:String,

        @Field(value="contrasena")
        contrasena:String

    ): Deferred<UsuarioResponse>

    @GET("usuario/{idusuario}")
    fun getUserInfo(

            @Path(value="idusuario")
            personalID:Int

    ): Deferred<UsuarioResponse>

}